include/RAJA/exec-cuda/forall_cuda.hxx

```diff
@@ -82,9 +82,29 @@ namespace RAJA
 //////////////////////////////////////////////////////////////////////
 //
 
-// HIDDEN namespace to encapsulate helper functions
-namespace HIDDEN
+// INTERNAL namespace to encapsulate helper functions
+namespace INTERNAL
 {
+
+/*!
+ ******************************************************************************
+ *
+ * \brief calculate global thread index from 1D grid of 1D blocks
+ *
+ ******************************************************************************
+ */
+__device__ __forceinline__ Index_type getGlobalIdx_1D_1D()
```

Comment:
I realize we had some of these before, but could you swap this out for a macro or something since forceinline may or may not be supported?